### PR TITLE
Update plot_time_series.py with content

### LINKED ISSUE #34: populate `plot_time_series` function contents

### PR DESCRIPTION
**Pull request by arturoboquin**

### CODE REVIEW

File: src/pyxplor/plot_time_series.py [hunk 1]
```diff
@@ -8,54 +10,71 @@ def plot_time_series(input_df: pd.DataFrame,
...
     super_title : str, optional
-        Super title for the entire plot. Default is "Time Series Analysis".
+        Title for the entire plot. Default is "Time Series Analysis".
 
     super_title_font : int, optional
         Font size for the super title. Default is 14.
 
     Returns
     -------
     None
```
Comment (phchen5) [inline src/pyxplor/plot_time_series.py]: I believe we're supposed to return the figure and axes object.

File: src/pyxplor/plot_time_series.py [hunk 2]
```diff
@@ -8,54 +10,71 @@ def plot_time_series(input_df: pd.DataFrame,
...
-    plot_time_series(data, 'date', ['sales', 'expenses'], freq='M', figsize=(12, 6))
     """
 
-    pass  # Implementation to be added
+    if not isinstance(input_df, pd.DataFrame):
+        raise ValueError("Input data must be a pandas DataFrame.")
+
+    if date_column not in input_df.columns:
+        raise ValueError(f"Date column '{date_column}' not found in DataFrame.")
+
```
Comment (phchen5) [inline src/pyxplor/plot_time_series.py:63]: What about checking for the other arguments?

File: src/pyxplor/plot_time_series.py [hunk 3]
```diff
@@ -8,54 +10,71 @@ def plot_time_series(input_df: pd.DataFrame,
...
+    input_df[date_column] = pd.to_datetime(input_df[date_column])
```
Comment (phchen5) [inline src/pyxplor/plot_time_series.py:64]: What if date_column is something that cannot be transformed into a datetime object?

File: src/pyxplor/plot_time_series.py (hunk 2 above)
Comment (arturoboquin) [reply src/pyxplor/plot_time_series.py:63]: I can add more tests

File: src/pyxplor/plot_time_series.py (hunk 3 above)
Comment (arturoboquin) [reply src/pyxplor/plot_time_series.py:64]: I can add more tests

File: src/pyxplor/plot_time_series.py (hunk 1 above)
Comment (arturoboquin) [reply src/pyxplor/plot_time_series.py]: I will check an example

**Review by rbouwer — APPROVED** [review]
LGTM!

**Pull request merged**